lib/DelegatedModuleFactoryPlugin.js

```diff
@@ -35,7 +35,8 @@ class DelegatedModuleFactoryPlugin {
 					const dependency = data.dependencies[0];
 					const request = dependency.request;
 					if (request && request.indexOf(scope + "/") === 0) {
-						const innerRequest = "." + request.substr(scope.length);
+						const innerRequest =
```

Comment:
It should try all mainFiles and also try to resolve without mainFile. Request ending with `/` should be handled too.

Reply:
What is the case when the request comes with "/" at the end?

I think I need some guidance on how to resolve multiple mainFiles because on the DelegateModuleFactory it stops when it resolves the first one.